io/disk/fiotest.py

```diff
@@ -59,7 +59,7 @@ def setUp(self):
         default_url = "https://brick.kernel.dk/snaps/fio-git-latest.tar.gz"
         url = self.params.get('fio_tool_url', default=default_url)
         self.disk = self.params.get('disk', default=None)
-        self.dir = self.params.get('dir', default='/mnt')
+        self.dir = self.params.get('dir', default=None)
```

Comment:
sorry  for the late review, this looks good as it can be YAML driven but what is the harm in having `/mnt` as default as we always have way override this via YAML ,  @AYUSHJAIN951 if you are using other set for yaml i suggest push those yaml in upstream

Reply:
@PraveenPenguin we have agreed to not have any default value for this parameter as it has other side effects when comes to block devices or raw disk. so I feel having `None` is fine